src/ralph/data_center/views.py

```diff
@@ -0,0 +1,23 @@
+from ralph.admin.views.extra import RalphDetailView
+
+
+class RelationsView(RalphDetailView):
+    icon = 'shekel'
+    label = 'Relations'
+    name = 'relations'
+    url_name = 'relations'
+    template_name = 'data_center/datacenterasset/relations.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        related_objects = {}
+
+        if self.object.cloudhost_set.exists():
+            related_objects['cloud_hosts'] = [
```

Comment:
why not `related_objects['cloud_hosts'] = list(self.object.cloudhost_set.all())`?

Reply:
Done